src/main/java/com/jagex/runescape/Game.java

```diff
@@ -11752,7 +11755,7 @@ private void method152() {
             boolean flag1 = false;
             try {
                 Buffer stream = SoundTrack.data(sound[index], soundType[index]);
-                new SoundPlayer(new ByteArrayInputStream(stream.buffer, 0, stream.currentPosition), soundVolume[index], soundDelay[index]);
+                new SoundPlayer(new ByteArrayInputStream(stream.buffer, 0, stream.currentPosition), soundType[index], soundDelay[index]);
```

Comment:
This was set incorrectly, causing sound volumes to not work. You can give sounds their own independent volume from 0-10 that works independently of what the user has selected within their sound settings. This change makes that part of the sound packet work again. :)